injectable.gemspec

```diff
@@ -20,10 +20,10 @@ Gem::Specification.new do |spec|
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
-  spec.required_ruby_version = '>= 2.3'
+  spec.required_ruby_version = '>= 2.5'
```

Comment:
This one is interesting. We don't have a rubocop file on the repo. So that file mentioned is being set up by CodeClimate itself? I can't find on that app how to set a different Rubocop version, not sure if that's even possible.

Perhaps we should consider dropping CC and move into HoundCI or some alternate offered by Github Actions.